packages/frontend/src/redux/slices/swap/index.js

```diff
@@ -6,6 +6,7 @@ import { batch } from 'react-redux';
 import CONFIG from '../../../config';
 import FungibleTokens from '../../../services/FungibleTokens';
 import fungibleTokenExchange from '../../../services/tokenExchange';
+import * as tokenUtils from '../../../utils/token';
```

Comment:
Generally prefer to "namespace" utility files, basically to the same variable name you've used here: `tokenUtils` or `TokenUtils` (since its a container of functionality, similar to a class).

Doing `import * as ...` is similar to default imports imo.

So you'd create a single `TokenUtils` object to export from the `token.ts` file, containing the functions.